src/components/home/Features.vue

```diff
@@ -0,0 +1,86 @@
+      <FeatureCard title="弹幕及弹幕云过滤">
+        <template #desc>
+          Animeko 从弹弹play, 其他弹幕网站, 以及 Animeko 弹幕服务获取弹幕，你也可以发送弹幕到 Animeko 的公益弹幕服务器，AI
+          团队将逐渐推进弹幕云过滤功能。
+        </template>
+        <template #default>
+          <svg
+            class="h-6 w-6 text-white" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor"
+            aria-hidden="true"
+          >
+            <path
+              stroke-linecap="round" stroke-linejoin="round"
+              d="M10.05 4.575a1.575 1.575 0 10-3.15 0v3m3.15-3v-1.5a1.575 1.575 0 013.15 0v1.5m-3.15 0l.075 5.925m3.075.75V4.575m0 0a1.575 1.575 0 013.15 0V15M6.9 7.575a1.575 1.575 0 10-3.15 0v8.175a6.75 6.75 0 006.75 6.75h2.018a5.25 5.25 0 003.712-1.538l1.732-1.732a5.25 5.25 0 001.538-3.712l.003-2.024a.668.668 0 01.198-.471 1.575 1.575 0 10-2.228-2.228 3.818 3.818 0 00-1.12 2.687M6.9 7.575V12m6.27 4.318A4.49 4.49 0 0116.35 15m.002 0h-.002"
+            />
+          </svg>
+        </template>
+      </FeatureCard>
+      <FeatureCard title="使用 bangumi 管理追番进度">
+        <template #desc>
+          只要有一个 bangumi 账号，你就可以通过 Animeko 一站式地管理追番的进度，标注想看/再看/看过的番剧，以及和网友们讨论剧集。当然，你也可以以游客的身份体验大部分功能。
+        </template>
+        <template #default>
+          <svg
+            class="h-6 w-6 text-white" xmlns="http://www.w3.org/2000/svg" width="1em" height="1em"
+            viewBox="0 0 24 24"
+          >
+            <g fill="none">
+              <path
+                stroke="currentColor" stroke-width="1.5"
+                d="M22 16c0 2.828 0 4.243-.879 5.121C20.243 22 18.828 22 16 22H8c-2.828 0-4.243 0-5.121-.879C2 20.243 2 18.828 2 16v-4c0-2.828 0-4.243.879-5.121C3.757 6 5.172 6 8 6h8c2.828 0 4.243 0 5.121.879C22 7.757 22 9.172 22 12z"
+              />
+              <path stroke="currentColor" stroke-linecap="round" stroke-width="1.5" d="m9 2l3 3.5L15 2" opacity="0.5" />
+              <path stroke="currentColor" stroke-linecap="round" stroke-width="1.5" d="M16 6v16" />
+              <path
+                fill="currentColor" d="M20 16a1 1 0 1 0-2 0a1 1 0 0 0 2 0m0-4a1 1 0 1 0-2 0a1 1 0 0 0 2 0"
+                opacity="0.5"
+              />
+            </g>
+          </svg>
+        </template>
+      </FeatureCard>
+      <FeatureCard title="代码开源">
+        <template #desc>
+          开放源代码, 公开自动构建, 无资料泄露风险，可 PR 添加自己喜欢的功能。
+        </template>
+        <template #default>
+          <svg
+            class="h-6 w-6 text-white" xmlns="http://www.w3.org/2000/svg" width="1em" height="1em"
+            viewBox="0 0 24 24"
+          >
+            <path
+              fill="currentColor"
+              d="M12 12h7c-.53 4.11-3.28 7.78-7 8.92zH5V6.3l7-3.11M12 1L3 5v6c0 5.55 3.84 10.73 9 12c5.16-1.27 9-6.45 9-12V5z"
+            />
+          </svg>
+        </template>
+      </FeatureCard>
+      <FeatureCard title="BT + 在线源播放">
+        <template #desc>
+          全自动整合多个 BT 数据源与在线数据源, 兼顾资源质量与缓冲速度，支持 emby 和 jellyfin 源，享受你的本地追番体验~
```

Comment:
```suggestion
          全自动整合多个 BT 数据源与在线数据源, 兼顾资源质量与缓冲速度。
```

Emby 和 Jellyfin 支持都比较差, 我觉得还是别提了

Reply:
6